test/upsert/rehydration/01-setup.td

```diff
@@ -9,12 +9,6 @@
 
 > DROP CLUSTER IF EXISTS storage_cluster CASCADE;
 
-> CREATE CLUSTER storage_cluster REPLICAS (
-    r1 (
-      STORAGECTL ADDRESSES ['clusterd1:2100'],
-      STORAGE ADDRESSES ['clusterd1:2103'],
-      COMPUTECTL ADDRESSES ['clusterd1:2101'],
-      COMPUTE ADDRESSES ['clusterd1:2102'],
-      WORKERS 4
-    )
-  )
+# Using mem-2 because it has defined memory limits which will be passed for
+# backpressure inflight bytes configuration
+> CREATE CLUSTER storage_cluster SIZE 'mem-2', REPLICATION FACTOR 1
```

Comment:
Is this fine? Otherwise, can also pass the memory limit directly to clusterd in mzcompose.

Reply:
I'm not sure myself if it was important for this test to be using the existing `clusterd1` container!

Reply:
Probably need @guswynn to weigh in on this one.

Reply:
yeah this has to be the existing clusterd1 container

Reply:
updated! passing the memory limit via clusterd args instead